simplq/src/components/pages/Home/StaticInfos.jsx

```diff
@@ -124,3 +124,9 @@ export const ExtraInfo = () => (
     </div>
   </div>
 );
+
+export const GetStarted = () => (
+  <div data-aos="fade-up" className={`${styles['section']} ${styles['get-started']}`}>
+    <h2 data-aos="zoom-in">Ready to get started with SimplQ?</h2>
```

Comment:
Can you add a button that on click will scroll all the way to the top? You can import and use this function: 

https://github.com/SimplQ/simplQ-frontend/blob/master/simplq/src/components/common/utilFns.js#L24